src/cluster_migrateslots.c

```diff
@@ -353,6 +358,151 @@ void fireModuleSlotMigrationEvent(slotMigrationJob *job, int subevent) {
     zfree(info.slot_ranges);
 }
 
+/* Encode active slot migrations into a string for storage into an RDB aux
+ * field.
+ *
+ * Format is:
+ * <job_name>:<source_node_name>:<start slot>-<end slot>( <start-slot2>-<end slot2>...)
+ *
+ * Multiple active slot migrations are joined together with "/"
+ */
+static sds clusterEncodeSlotImportsRDBAuxField(int rdbflags) {
```

Comment:
Not required in this PR, but it would be good to have a test that loads an RDB with pending slot migrations and makes sure the data from the in-progress migrations is dropped.